gzipstream/gzipstreamfile.py

```diff
@@ -0,0 +1,89 @@
+import io
```

Comment:
We definitely want to attribute and give credit to the original author / repository / license here!